schema/individual.json

```diff
@@ -118,45 +118,37 @@
       "minItems": 1,
       "description": "List of polyfill urls"
     },
-    "implementations": {
-      "type": "array",
-      "items": {
-        "type": "string",
-        "enum": [
-          "Babel",
-          "ChakraCore",
-          "Chrome",
-          "Edge",
-          "Engine262",
-          "Firefox",
-          "GraalJS",
-          "Hermes",
-          "JavaScriptCore",
-          "Nashorn",
-          "Node",
-          "QuickJS",
-          "Safari",
-          "SpiderMonkey",
-          "TypeScript",
-          "v8"
-        ]
-      },
-      "minItems": 1,
-      "uniqueItems": true,
-      "additionalItems": false,
-      "description": "List of implementations"
+    "support": {
+      "allOf": [
+        {
+          "$ref": "https://raw.githubusercontent.com/mdn/browser-compat-data/master/schemas/compat-data.schema.json#/definitions/support_block",
+          "description": "MDN Support data definition"
+        },
+        {
+          "type": "object",
+          "propertyNames": {
+            "enum": ["babel", "chakracore", "engine262", "graaljs", "hermes", "javascriptcore", "nashorn", "quickjs", "spidermonkey", "typescript", "v8"]
+          },
+          "additionalProperties": {
+            "$ref": "https://raw.githubusercontent.com/mdn/browser-compat-data/master/schemas/compat-data.schema.json#/definitions/support_statement"
+          },
+          "description": "TC39 Support data definition (JavaScript engine)"
```

Comment:
@codehag i extend definition range, allow input javascript engine in the support data.

Reply:
Right, I see -- because mdn isn't quite so concerned with things like engine262. 

That makes sense.